lib/genid.js

```diff
@@ -21,7 +21,7 @@ module.exports = exports = function() {
     };
 
     function getRandomChar(array) {
-        var rNum = Math.floor(Math.random() * (array.length -1));
+        var rNum = Math.floor(Math.random() * array.length);
```

Comment:
`Math.random()` is inclusive of 0, but exclusive of 1. So combined with `Math.floor()` we'll get a random number `0` through `array.length-1`, no need for the explicit `-1`.